js/tools/rainbow.js

```diff
@@ -0,0 +1,40 @@
+/* global app */
+window.tools.rainbow = {
+  name: 'rainbow',
+  icon: '/images/rainbow.png',
+  state: {
+    selected: false,
+    mousePressed: false,
+    prevMouse: { x: null, y: null }
+  },
+  getRandomColor: function () {
+    return { g: Math.floor(Math.random() * 256), b: Math.floor(Math.random() * 256) }
+  },
```

Comment:
assuming u're trying to return an object with a `r`, `g` && `b` properties, but this one is missing the `r`

Reply:
actually, I just want a random base g and b blue. I vary the red in the brush. But I guess I should make it more clear in the function.